app/agents/voice/automatic/tools/breeze/configuration.py

```
@@ -278,60 +239,101 @@ async def _handle_create_surcharge_rule(params: FunctionCallParams):
+        headers["Authorization"] = AWS_VAYU_WRITE_API_KEY
+
+        for rule in rules_to_delete:
+            if existing_rule_id := rule.get("id"):
+                try:
+                    delete_endpoint = (
+                        f"{AWS_VAYU_URL}/surcharge/rule?ruleId={existing_rule_id}"
+                    )
+                    async with create_http_client() as client:
+                        await client.delete(delete_endpoint, headers=headers)
+                    deleted_count += 1
+                except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)